cfme/containers/node.py

```diff
@@ -190,5 +190,13 @@ def step(self):
         """Navigate to the Timelines page"""
         self.prerequisite_view.toolbar.monitoring.item_select('Timelines')
 
-# TODO Need Ad hoc Metrics
+
+@navigator.register(Node, 'AdHoc')
```

Comment:
**Question**: Thoughts on making this ```AdHocMetrics```? I think being more specific here would be better. I know it would also have to change for https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/provider/__init__.py#L472